cypress/e2e/app/accountDetail.cy.js

```diff
@@ -3,15 +3,15 @@ describe('account detail', () => {
     cy.visit(`/accounts/${Cypress.env('accountAddress')}`)
 
     cy.get('.account-details-panel').should('be.visible')
-    cy.get('.account-activities-panel table').should('be.visible')
+    cy.get('.account-activities-panel .paginated-content').should('be.visible')
```

Comment:
Consequently, I fixed the tests to pass. It's not related to the issue but also has a priority. Previously the selector was dependent on data that are now missing, therefor the test would fail also on another PRs